src/api/entities/CheckpointSchedule/types.ts

```diff
@@ -1,16 +1,6 @@
 import BigNumber from 'bignumber.js';
 
-import { CalendarPeriod } from '~/types';
-
-export type ScheduleDetails = {
+export interface ScheduleWithDetails {
   remainingCheckpoints: BigNumber;
   nextCheckpointDate: Date;
-};
-
-export type ScheduleParams = {
```

Comment:
I didn't say to move this to `~/types`. You had to do as every other Entity does:

- Export the interface from this file
- Export all CheckpointSchedule types from `src/api/entities/types.ts` (while you're at it, please define and export a `CheckpointSchedule` type in the same way it's being done for other entities)
- Whenever you want to import these types, you import them from `~/types`